README.md

```diff
@@ -1 +1,46 @@
-# tech-academy
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Tech Academy
+
+An online website for a Tech academy that show the available courses and the top instructors   
+
+
+## Built With
+
+- HTML ,CSS, JavaScript
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple steps.
+
+git clone git@github.com:Zainab-Alzubair/tech-academy.git
+
+## Online version
+https://zainab-alzubair.github.io/tech-academy/
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/Zainab-Alzubair)
+- Twitter: [@twitterhandle](@zainabm34401029)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/zainab-al-zubair-bb6777168/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```

Comment:
- [ ] You didn't gave any credit to the person whom design you have used. You can do something similar which i did below :+1:

```
## Acknowledgments
Original design is from [Cindy Shin](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).
```